man/cards-package.Rd

```diff
@@ -6,6 +6,8 @@
 \alias{cards-package}
 \title{cards: Analysis Results Data}
 \description{
+\if{html}{\figure{logo.png}{options: style='float: right' alt='logo' width='120'}}
+
 Construct CDISC (Clinical Data Interchange Standards Consortium) compliant Analysis Results Data objects. These objects are used and re-used to construct summary tables, visualizations, and written reports. The package also exports utilities for working with these objects and creating new Analysis Results Data objects.
```

Comment:
I'm not sure how this line got changed since it's not in your previous PR, but I'll revert it since it's unrelated.

```suggestion
Construct CDISC (Clinical Data Interchange Standards Consortium) compliant Analysis Results Data objects. These objects are used and re-used to construct summary tables, visualizations, and written reports. The package also exports utilities for working with these objects and creating new Analysis Results Data objects.
```